i18n/en.json

```diff
@@ -2785,6 +2786,9 @@
   "user.settings.import_theme.submitError": "Invalid format, please try copying and pasting in again.",
   "user.settings.languages.change": "Change interface language",
   "user.settings.languages.promote": "Select which language Mattermost displays in the user interface.<br /><br />Would like to help with translations? Join the <a href='http://translate.mattermost.com/' target='_blank'>Mattermost Translation Server</a> to contribute.",
+  "user.settings.timezones.automatic": "Set automatically",
+  "user.settings.timezones.change": "Change timezone",
+  "user.settings.timezones.promote": "Select the time zone used for timestamps in the user interface and email notifications.",
```

Comment:
`time zone` vs. `timezone`? I think they are both fungible, but perhaps we converge on one usage?

Reply:
Nice catch, I'll fix that.  We can go with timezone to be consistent